environment.yml

```diff
@@ -55,6 +55,7 @@ dependencies:
     - folium
     - geopy
     - otter-grader==3.1.4
+    - tensorflow
```

Comment:
Let's pin a version here?

I like pip for tensorflow as it's what they recommend doing on their website.